lib/easybill-rest-client/api_client.rb

```diff
@@ -26,55 +29,58 @@ def request_collection(method, endpoint, params = {})
     end
 
     def request(method, endpoint, params = {})
-      Retryable.retryable(
-        tries: tries,
-        sleep: retry_cool_off_time,
-        on: EasybillRestClient::TooManyRequests
-      ) do
-        response = perform_request(method, endpoint, params)
-        process_response(response)
+      @request_logger = RequestLogger.new(logger: logger, request_id: Time.now.to_f)
```

Comment:
I was thinking that the timestamp might be faster. I would mainly search for it by copying the ID, not visually. But I can go with the hex string as well, if you think it's better.